rs/index/src/utils.rs

```diff
@@ -33,3 +34,31 @@ impl SearchContext {
         self.visited_pages.as_ref().unwrap().len()
     }
 }
+
+#[derive(Debug)]
+pub struct IdWithScore {
+    pub id: u64,
+    pub score: f32,
+}
+
+impl Ord for IdWithScore {
```

Comment:
also what happens if it's a tie? do we want to use `id` as tie breaker?

Reply:
Yeah it's in `PartialEq`